fplll/test.cpp

```diff
@@ -19,17 +19,47 @@
 using namespace std;
 using namespace fplll;
 
+
```

Comment:
Hum. Removing that file seems to have been a bad idea. make distr now
fails. I guess there is something to remove from a config file ?

2016-06-30 18:21 GMT+02:00 Martin Albrecht <notifications@github.com>:

> In fplll/test.cpp
> <https://github.com/fplll/fplll/pull/114#discussion_r69164257>:
>
> > @@ -19,17 +19,47 @@
> >  using namespace std;
> >  using namespace fplll;
> >
> > +
>
> Is editing test.cpp a mistake? (I'd guess this file should be removed, in
> general)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fplll/fplll/pull/114/files/653e012a80e0c6a18bcb3bf21535b25acb9062c3#r69164257>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGBp4M8aHSQsrxcQ0WDIOun5JyX8Yyp9ks5qQ-0hgaJpZM4I9z22>
> .
>